include/pref_colors.hpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of the QPxTool project.
+ * Copyright (C) 2008-2009 Gennady "ShultZ" Kozlov <qpxtool@mail.ru>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ * See the file "COPYING" for the exact licensing terms.
+ * */
+
+#ifndef PREF_COLORS_H
+#define PREF_COLORS_H
+
+#include "qpxsettings.hpp"
+#include <QWidget>
+
+class QBoxLayout;
+class QGridLayout;
+class QPushButton;
+class QMenu;
+class QAction;
+class QLineEdit;
+
+class ColorLabel;
+
+class prefColors : public QWidget {
+  Q_OBJECT
+public:
+  prefColors(QPxSettings *iset, QWidget *p = 0, Qt::WindowFlags fl = 0);
```

Comment:
Codacy found an issue: [Class 'prefColors' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6191247)